### ISSUE TITLE
How to use  Universal Dependencies style ?

### ISSUE THREAD

**Issue by Lix1993** [bminixhofer/wtpsplit#108]
I load model from local path,:

```
from wtpsplit import WtP
wtp = WtP("/data/share/HuggingFace/custom/benjamin/wtp-bert-mini/")

wtp.split("This is a test This is another test.", lang_code="en", style="ud")
```

it returns :
ValueError: This model does not have any associated mixtures. Maybe they are missing from the model directory?

where can I download associated mixtures?

**Comment by bminixhofer** [comment]
Hi, downloading this file: https://huggingface.co/benjamin/wtp-bert-mini/blob/main/mixtures.skops to the model directory should to the trick.

**Comment by Lix1993** [comment]
hi @bminixhofer , it doesn't work.

I download the model using git:

```
git lfs clone https://huggingface.co/benjamin/wtp-bert-mini
WARNING: 'git lfs clone' is deprecated and will not be updated
          with new flags from 'git clone'

'git clone' has been updated in upstream Git to have comparable
speeds to 'git lfs clone'.
Cloning into 'wtp-bert-mini'...
remote: Enumerating objects: 29, done.
remote: Counting objects: 100% (6/6), done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 29 (delta 1), reused 0 (delta 0), pack-reused 23
Unpacking objects: 100% (29/29), 5.08 KiB | 400.00 KiB/s, done.
Downloading LFS objects: 100% (3/3), 47 MB | 7.2 MB/s

ll wtp-bert-mini/
total 45696
drwxrwxr-x  3 lixing lixing     4096 Oct 18 09:21 ./
drwxr-xr-x 13 lixing   4000     4096 Oct 18 09:21 ../
-rw-rw-r--  1 lixing lixing 29577694 Oct 18 09:21 pytorch_model.bin
drwxrwxr-x  9 lixing lixing     4096 Oct 18 09:21 .git/
-rw-rw-r--  1 lixing lixing     1528 Oct 18 09:21 .gitattributes
-rw-rw-r--  1 lixing lixing     6139 Oct 18 09:21 config.json
-rw-rw-r--  1 lixing lixing      549 Oct 18 09:21 README.md
-rw-rw-r--  1 lixing lixing 14800980 Oct 18 09:21 model.onnx
-rw-rw-r--  1 lixing lixing  2377183 Oct 18 09:21 mixtures.skops
```


[mixtures.skops](https://huggingface.co/benjamin/wtp-bert-mini/blob/main/mixtures.skops) this file is downloaded.

then loading model from local path:

```
from wtpsplit import WtP
wtp = WtP("/home/lixing/wtp-bert-mini/")
wtp.split("This is a test This is another test.", lang_code="en", style="ud")
```
it failed.

**Comment by bminixhofer** [comment]
Hi, sorry, this should be fixed in version 1.2.4, see #110.
